kubernetes-deploy.gemspec

```diff
@@ -29,6 +29,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency "ejson", "~> 1.0"
   spec.add_dependency "colorize", "~> 0.8"
   spec.add_dependency "statsd-instrument", "~> 2.2"
+  spec.add_dependency "jsonpath", "~> 0.9.6"
```

Comment:
## Pros
- Checked for jsonpath CVE vulnerabilities, found none.
- Examined source code for cases of `eval`. `eval` not used in the gem since version `0.8.4` (currently using `0.9.8`).
- Rubygems records 4 million+ downloads of gem, ~200k for current version.

## Cons
- Repo not very active
- Poor error handling

## Neutral
- No obvious alternative jsonpath implementation (`jq` maybe?)

Reply:
`jq` has other problems, it requires a native library and then proceeds to pollute your `stdout` since it wasn't really designed to work as a library.